exercism/wordy/wordy.js

```diff
@@ -7,33 +7,42 @@ module.exports = {
         'minus': '-',
         'multiplied': '*',
         'divided': '/'
-      },
-      this.sequence = []
-
+      }
     }
 
-    answer() {
-      let splitString = this.input.replace('?',"")
+    separateString( data ) {
+      return data.replace('?',"")
       .replace(/by/g,'').split(' ').slice(2)
```

Comment:
JRob has recommended to me that when you chain functions on a new line like this, it's good practice to tab the next line so it's clear that it belongs to the above line.